### ISSUE TITLE
Add to the awesome repo?

### ISSUE THREAD

**Issue by erictleung** [danielecook/Awesome-Bioinformatics#2]
Hi @danielecook,

Have you considered trying to add this repo to the [awesome repo list](https://github.com/sindresorhus/awesome) along with the other awesome repos? I think it'd be a great addition.

**Comment by danielecook** [comment]
Sure! I'll give it a try.

**Comment by erictleung** [comment]
Cool! Great work on the list so far.

I was trying to think of other awesome things bioinformatics I could add and thought of the repository, [Informatics for RNA-seq: A web resource for analysis on the cloud](https://github.com/griffithlab/rnaseq_tutorial). It's not a tool, per se, but it is an educational resource on performing RNA-seq analysis using Amazon's AWS cloud services. Another awesome resource I know of is [RNA-seqlopedia](http://rnaseq.uoregon.edu/). The description for it is:

> "The RNA-seqlopedia provides an overview of RNA-seq and of the choices necessary to carry out a successful RNA-seq experiment."

tl;dr Would you be open to adding (educational) bioinformatics resources to the list? I understand if you wish to keep the scope of the repo to libraries and software. Just trying to add to the awesomeness :)

**Comment by danielecook** [comment]
Definitely! The only caveat is that I'd like to keep the quality top notch. Both of those links look great. 

Here are some of my favorites.

* [Rosalind](Rosalind.info)
* [Genomics Papers: Leek Group](https://github.com/jtleek/genomicspapers/)

I think its best to keep tools and educational sections separate. Within the educational sections we can break it out by method (e.g. RNA-Seq, DNA-Seq, ChIP-Seq, Mass Spec, etc.).

Feel free to submit a pull request. I'll add in some resources too.
